src/holofuel/pages/Profile/Profile.js

```diff
@@ -32,6 +42,9 @@ export default function Profile () {
         query: HolofuelUserQuery
       }]
     })
+      .catch(() => setOptimisticNickname())
```

Comment:
@JettTech one other addition for your attention, here I'm catching any potential error from the update call. If it throws an error, we reset the optimisticNickname to what it was on page load.